io.github.lxi-tools.yaml

```diff
@@ -0,0 +1,142 @@
+---
+app-id: io.github.lxi-tools
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: lxi-gui
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=host"
+modules:
+  - name: lua
+    sources:
+      - type: archive
+        url: https://www.lua.org/ftp/lua-5.3.6.tar.gz
+        sha256: fc5fd69bb8736323f026672b1b7235da613d7177e72558893a0bdcd320466d60
+      - type: patch
+        path: lua-makefile-is-shit.patch
```

Comment:
maybe a less offensive patch name?

Reply:
Ha ha - sorry about that!

When I named the patch I was furious because I had just found that that the lua Makefile STILL does not support building lua as a shared library. That is a problem I ran into more than a decade ago and still lua upstream has not be able to fix this despite the fact that literally every single Linux distribution out there needs to build and distribute lua as a shared library and consequently everyone carries around these stupid hacky lua makefile patches forever and now with the rise of Meson also hacky lua pkg-config files. It's all just very silly.

Reply:
Fixed.

Reply:
well you could just use the shared-modules.

https://github.com/flathub/shared-modules/